src/components/backup/new.vue

```diff
@@ -23,9 +23,9 @@ except according to the terms contained in the LICENSE file.
         </p>
         <form @submit.prevent="initiate">
           <label class="form-group">
-            <input ref="passphrase" v-model.trim="passphrase"
+            <input ref="passphrase" v-model="passphrase" type="password"
```

Comment:
Make this `type="text"` with `autocomplete="off"`. Do the same thing for the enable encryption modal.